plugins/services/src/js/reducers/serviceForm/JSONReducers/EnvironmentVariables.js

```diff
@@ -59,11 +59,13 @@ module.exports = {
   },
 
   JSONParser(state) {
-    if (state.env == null) {
+    if (state.env == null && state.environment == null) {
       return [];
     }
 
-    return Object.keys(state.env).reduce(function(memo, key, index) {
+    const env = state.env || state.environment;
```

Comment:
This is used for both multi-container and single-container services, and environment variables for multi-containers use the `environment` key while single-container services use the `env` key.